workflow/scripts/calc-precision-recall.py

```
@@ -59,45 +68,50 @@ def register(self, record, truth, query):
+                (self.tp_truth, self.tp_truth_vaf) = self.increment_counter(record, truth, self.tp_truth, self.tp_truth_vaf)
             elif c.cls is Class.TP_query:
-                self.tp_query = self.increment_counter(record, truth, self.tp_query)
+                (self.tp_query, self.tp_query_vaf) = self.increment_counter(record, truth, self.tp_query, self.tp_query_vaf)
             elif c.cls is Class.FN:
-                self.fn = self.increment_counter(record, truth, self.fn)
+                (self.fn, self.fn_vaf) = self.increment_counter(record, truth, self.fn, self.fn_vaf)
             elif c.cls is Class.FP:
-                self.fp = self.increment_counter(record, query, self.fp, True)
+                (self.fp, self.fp_vaf) = self.increment_counter(record, query, self.fp, self.fp_vaf, True)
             else:
                 assert False, "unexpected case"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)